test/TEST-16-ISSUE-518/avocado.conf

```diff
@@ -0,0 +1,2 @@
+[datadir.paths]
+logs_dir = /tmp/job-results
```

Comment:
> Why do we need avocado-framework?

We don't necessarily.  But I think that framework is actually quite nice for writing high-level "black box" tests. It would be nice to agree (or at least advise to people) on the test framework if they need to write "integration test" for systemd. Anyway, I wrote test mostly for myself, in order to make sure that patch looks as intended. We don't have to include it upstream.

> This test doesn't run tests inside VM/container (it uses the installed systemd, not the one from the built source tree). Why should we place this test in test/TEST-*-ISSUE-*?

I don't follow? AFAICT, there are no other "issue specific" tests anywhere else in systemd source tree, except test/. This test exercises new code added to fix issue #518 so I added it to test/ directory.

Yes it will use systemd from the system but I don't think it is a job of the test case to prepare test environment.